src/neo-vm/ExecutionEngine.cs

```diff
@@ -25,9 +25,9 @@ public class ExecutionEngine : IDisposable
         public virtual int Min_SHL_SHR => -ushort.MaxValue;
 
         /// <summary>
-        /// Set the max size in bytes allowed size for BigInteger
+        /// The max size in bytes allowed size for BigInteger
         /// </summary>
-        public virtual int MaxSizeForBigInteger => 32;
+        public const int MaxSizeForBigInteger = 32;
```

Comment:
`const` ? maybe we should use other constant, because this is virtual to allow the inheritance in neo

Reply:
It is important to ensure that `ByteArray` has the same behavior with `Integer`. I think it's ok for `MaxSizeForBigInteger` to be a `const`.